specification/search/data-plane/Azure.Search/preview/2024-03-01-Preview/examples/SearchIndexSearchDocumentsPost.json

```
@@ -110,7 +111,7 @@
             "title": "Fancy Hotel"
           }
         ],
-        "@odata.nextLink": "https://myservice.search.windows.net/indexes('myindex')/docs/search.post.search?api-version=2024-03-01-Preview"
+        "@odata.nextLink": "https://myservice.search.windows.net/indexes('myindex')/docs/search.post.search?api-version=2023-10-01-Preview"
```

nit, this api version is wrong i think

```suggestion
        "@odata.nextLink": "https://myservice.search.windows.net/indexes('myindex')/docs/search.post.search?api-version=2024-03-01-Preview"
```